.azure/azure-pipelines.yml

```diff
@@ -50,9 +53,6 @@ stages:
 
       - job: Build_And_Test
         displayName: Build And Test
```

Comment:
This should be just "build" now

Reply:
I left the job name as "build and test" because it's a unique id and would invalidate certain GH configurations for branch protection.

Reply:
There are only two of us who can merge to the protected branches, so you could just remove the requirement for the "Build and Test" job and add a requirement for the "Build" job after this is merged

Reply:
Agreed, but I'll do it BC as a solo commit after this is merged.

Reply:
what does BC mean?

Reply:
Backward Compatible. This PR doesn't alter any configuration changes in Github. A future PR can make that backwards-incompatible alteration.

Reply:
Oh, that's what backwards compatible means.